docs/api.rst

```diff
@@ -3114,35 +3114,45 @@ Get asset identifiers mappings
       Content-Type: application/json
 
       {
-          "result": {
-              "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31": {
-                  "name": "0xBitcoin",
-                  "symbol": "0xBTC",
-                  "asset_type": "evm token"
+        "result":{
+            "assets":{
+              "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31":{
+                  "name":"0xBitcoin",
+                  "symbol":"0xBTC",
+                  "asset_type":"evm token",
+                  "collection_id": "0"
               },
-              "DCR": {
-                  "name": "Decred",
-                  "symbol": "DCR",
-                  "asset_type": "own chain"
+              "DCR":{
+                  "name":"Decred",
+                  "symbol":"DCR",
+                  "asset_type":"own chain"
               },
-              "eip155:1/erc20:0xcC4eF9EEAF656aC1a2Ab886743E98e97E090ed38": {
-                  "name": "DigitalDevelopersFund",
-                  "symbol": "DDF",
-                  "chain_id": 1,
-                  "is_custom_asset": false,
-		  "asset_type": "evm token"
+              "eip155:1/erc20:0xcC4eF9EEAF656aC1a2Ab886743E98e97E090ed38":{
+                  "name":"DigitalDevelopersFund",
+                  "symbol":"DDF",
+                  "chain_id":1,
```

Comment:
why did you remove the styling here? Please undo and make it like the rest of the file with a space between `:` and the other stuff. In all of the json